core/model/src/commonMain/kotlin/io/github/droidkaigi/confsched2023/model/AppError.kt

```diff
@@ -1,7 +1,8 @@
 package io.github.droidkaigi.confsched2023.model
 
 public sealed class AppError : RuntimeException {
-    private constructor(message: String? = null, cause: Throwable? = null) : super(message, cause)
+    private constructor(message: String?, cause: Throwable?) : super(message, cause)
```

Comment:
I have kept this constructor considering the possibility that we might want to set a fixed message in the future. 
Please let me know in the comments if you think it should be removed :pray: